torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -154,3 +154,21 @@ def convert_image_color_space_pil(
         return image
 
     return image.convert(new_mode)
+
+
+def convert_color_space(
```

Comment:
I think we botched the naming of the other kernels a little. It should be `convert_color_space_image_{tensor, pil}` rather than `convert_image_color_space_{tensor, pil}`. Naively following the old scheme would give us `convert_image_color_space` for the dispatcher, which is misleading since all inputs are allowed.

Reply:
Nothing, just wanted confirmation to do it.

Reply:
It's a good idea. Let's do it.